Firmware/MVP/battsci.cpp

```diff
@@ -117,60 +294,60 @@ void BATTSCI_sendFrames()
       frameSum_87 += BATTSCI_writeByte( 0x40 );                                           //Never changes
       frameSum_87 += BATTSCI_writeByte( (spoofedVoltageToSend >> 1) );                    //Half Vbatt (e.g. 0x40 = d64 = 128 V)
 
-      if(LTC68042result_loCellVoltage_get() <= 30000 )
-      { //at least one cell is severely under-charged.  Disable Assist.
-        frameSum_87 += BATTSCI_writeByte( 0x11 );                                         //Battery SoC (upper byte)
-        frameSum_87 += BATTSCI_writeByte( 0x48 ); //20% SoC                               //Battery SoC (lower byte)
+      LiBCM_SoC = SoC_getBatteryStateNow_percent();
+      if(LiBCM_SoC <= 5 )
+      {
+        //at least one cell is severely under-charged.  Disable Assist.  0x11 0x48 is 20% SoC.  This if statement functionally performs a negaive recal.
+		//turns out 0x20 0x48 is ALSO 20% SoC
+        SoC_Bytes[0] = 0x20;
+        SoC_Bytes[1] = 0x48;
+
+        // Make sure SoC doesn't immediately spike back up
+        calculatedSoC = 0;
+        oldCalculatedSoC = 0;
+        temperature_Byte = 0x30;
+        SoCUpdateDelayCounter = 0;  // Setting this to 0 keeps SoC at 20% for extra frames so the IMA has extra time to be charged.
```

Comment:
We should implement hysteresis differently.  Rather than wait N loops through the code, we should define a hysteresis SoC percentage (e.g. 5%), and then toggle between a boolean.  Example:
`#define SoC_HYSTERESIS_PERCENT 5
#define FAVOR_ASSIST true
#define FAVOR_REGEN false
bool SoC_mode = FAVOR_ASSIST;`

and then in our BATTSCI regen/assist flag code, when we transition from one SoC range to the next we say:
```
if(SoC < (25% + (boolSoC_mode & SoC_HYSTERESIS_PERCENT) )
{
   //disable assist case
   //send flags, etc... whatever the existing code is
  bool SoC_mode = FAVOR_REGEN;
}

...

if(SoC > 60% + (boolSoC_mode & SoC_HYSTERESIS_PERCENT) )
{
  //limit regen case
  //existing code
  bool SoC_mode = FAVOR_ASSIST;
}
```